### DISCUSSION TITLE
Vue table

### DISCUSSION THREAD

**Discussion by esag-admin** [esag-swiss/iDig-Webapp#3] [Ideas]
Je me demande si on n devrait pas upgrader le module Vue de gestion de table pour avoir plus de fonctionnalités, en particulier "resize and drag-and-drop column". Ou c'est un trop gros travail de de migration?

https://happy-coding-clans.github.io/vue-easytable/#/en/demo
https://www.ag-grid.com/vue-data-grid/
https://www.jqwidgets.com/jquery-widgets-demo/demos/jqxgrid/index.htm

Drag-and-drop: https://sortablejs.github.io/vue.draggable.next/#/table-column-example

**Comment by AlZorba** [comment]
L'ajout d'un complément est en effet indiqué. Après un rapide coup d'oeil, ceux proposés me semble difficilement intégrable. Une orientation possible est Vuetify ou Quasar.

**Reply by francoiscrouzet** [reply]
@AlZorba regarde tout ce qu'il ya comme fonctionnalité : https://quasar.dev/vue-components/table
export CSV, loader, no data, sort, responsive, popup edit ...
